### ISSUE TITLE
Doc improvement suggestion: extra settings needed if using axios v1.5.0 in vitest/jest

### ISSUE THREAD

**Issue by zoltan-nz** [kettanaito/msw#1747]
### Scope

Improves an existing behavior

### Compatibility

- [ ] This is a breaking change

### Feature description

Axios' latest version changed the underlying adapter configuration.

Using `axios` `v1.5.0` with `msw` needs an extra line of code in the test setup.

For example: `setup-tests.ts`

```typescript
import axios from 'axios'
import { afterAll, afterEach, beforeAll } from 'vitest'
import { server } from './mocks/server'

beforeAll(() => {
  axios.defaults.adapter = 'xhr'
  server.listen({ onUnhandledRequest: 'warn' })
})

afterEach(() => server.resetHandlers())
afterAll(() => server.close())
```
Have to use `axios.defaults.adapter = 'xhr'`. Otherwise, `msw` interception would not work.

Source: https://github.com/axios/axios/issues/5844

**Comment by kettanaito** [comment]
Hi, @zoltan-nz. Thanks for raising this. 

MSW should intercept Axios requests regardless which adapter you're using. Can you please submit a reproduction repository when it fails to do so? We've got some integration tests with Axios, I'd love to add your use case to them to prevent issues like this. Thanks.

**Comment by zoltan-nz** [comment]
@kettanaito Thank you! Meanwhile, Axios fixed the issue, and with `axios` `v1.5.1` everything is back to normal. ;)

https://github.com/axios/axios/pull/5837
